docs/program/extend/_index.md

```diff
@@ -20,5 +20,5 @@ You can extend Viam in these and other ways by creating and using custom resourc
 
 {{< cards >}}
```

Comment:
I think this page needs to provide guidance on whether to use custom components or modular resources

Reply:
Also this snippet of text previously was quite good for context I think

```
The Viam platform manages:

- Starting and stopping the module binary (lifecycle)
- Dependency management (modular resources may depend on other resources - both RDK built-in and other modular resources)
- Securely exposing the modular resource's interface ({{< glossary_tooltip term_id="protobuf" text="Protobuf" >}}-described API) alongside Viam RDK built-in APIs
- Robot reconfiguration for any configured modular resource instances
```

Reply:
I added a note saying modular resources was supported unless you have a non-linux platform or another issue, which is what James suggested
I have moved that info to the Modular Resource page (/program/extend/modular-resources/#modular-resource-management) but I don't think it needs to be intro level on the whole extend page, LMK if you would still prefer to add

Reply:
true - I somehow forgot that you added that note.
And good point - ok this all looks good to me.